diff --git a/flymake-posframe.el b/flymake-posframe.el
--- a/flymake-posframe.el
+++ b/flymake-posframe.el
@@ -94,6 +94,7 @@
 	   :internal-border-width 3
 	   :left-fringe 1
 	   :right-fringe 1
+	   :alpha-backaground 100
 	   :foreground-color (face-foreground 'flymake-posframe-foreground-face nil t)
 	   :background-color (face-background 'flymake-posframe-background-face nil t)
            :string (concat (propertize

Diff finished.  Thu Mar 28 12:12:02 2024